web/src/components/pages/contribution/contribution.css

```diff
@@ -357,13 +353,13 @@
     }
 
     .primary-buttons {
-        margin-top: 2vh;
+        margin-top: 6vh;
         display: flex;
         justify-content: center;
         align-items: center;
 
-        @media (--xl-up) {
-            margin-top: 12vh;
+        @media (min-height: 910px) {
```

Comment:
I suggest we create a custom media query for this as we do in [media.css](https://github.com/common-voice/common-voice/blob/main/web/src/components/media.css) so we can keep track of the media queries we use in the application.

We can use the name - `xl-tall` for this value